flink-jepsen/src/jepsen/flink/db.clj

```diff
@@ -108,7 +108,12 @@
     (teardown! [_ test node]
       (c/su
         (try
-          (doseq [db (reverse dbs)] (db/teardown! db test node))
+          (doseq [db (reverse dbs)]
+            (try
+              (db/teardown! db test node)
+              ;; jepsen also calls teardown at the start of the test
+              ;; neither our dbs nor jepsen-zookeeper handle the db not existing gracefully
```

Comment:
How is Flink failing when calling teardown w/o having it started first?

Reply:
* Flink DB doesn't fail (because Flink is extracted somewhere earlier)
* Zookeeper DB fails because it tries to stop a service which doesn't exist, returning a non-zero exit code
* Hadoop DB fails because it tries to invoke a script which doesn't exist
* Kafka DB should also be fine; I don't recall it throwing errors

Reply:
There are certainly cleaner ways to fix it, but this already took way too much time. (The incompatibility with Clojure 1.10.1+ caught me off-guard, and I kept trying to find some API change in Jepsen 😢 )

Reply:
Ok, let's be pragmatic here.